libafl/src/feedbacks/map.rs

```diff
@@ -21,6 +21,9 @@ use crate::{
     Error,
 };
 
+/// A [`MapFeedback`] that implements the afl algorithm
+pub type AflMapFeedback<FT, I, O, S, T> = MapFeedback<FT, I, MapAflFilter, O, AflReducer, S, T>;
```

Comment:
I don't get why this, it is already implemented in the observer to avoid to run it for imported inputs.
https://github.com/AFLplusplus/LibAFL/blob/main/libafl/src/observers/map.rs#L467

Reply:
I didnt know it is there. I thought it is missing. you remember we talked about this.
so is this in the observer always used? or only if this is requested by the user (special observer?). what is then missing to have afl-like behaviour instead of only keeping the highest value?

Reply:
Yes, that, hitcounts are already in

Reply:
Would it be slower to have this as feedback? I think it's easier to understand than in the observer(?)

Reply:
No it's not easier, and preprocessing the map for bucketing is an expensive task. Doing it in the observer means that we serialize a preprocessed map for the other node, avoiding this operation on imported inputs

Reply:
what is faster is better, and I already woke up this morning realizing that using a map is required and faster and already wrote to dominik about it before looking into here :)
but - coverage is needed on imported inputs too. or why not?

Reply:
Because, when the configuration of two nodes matches, we share the input and the observer. The receiver must only call it's feedback to know if the received input is interesting. Doing the count class preprocessing in the sender means that all the other receivers can avoid to do it, so it's faster.